### PR TITLE
Set audit directory to be 0700

### PR DESCRIPTION
**Pull request by soltysh**
/assign @mfojtik @sttts

### CODE REVIEW

File: bindata/v3.11.0/openshift-apiserver/ds.yaml [hunk 1]
```diff
@@ -26,9 +26,12 @@ spec:
...
         image: ${IMAGE}
         imagePullPolicy: IfNotPresent
-        command: ["hypershift", "openshift-apiserver"]
+        command: ["/bin/bash", "-xec"]
         args:
-        - "--config=/var/run/configmaps/config/config.yaml"
+        - |
+          mkdir -p /var/log/openshift-apiserver
+          chmod 0700 /var/log/openshift-apiserver
+          exec hypershift openshift-apiserver --config=/var/run/configmaps/config/config.yaml
```
Comment (sttts) [inline bindata/v3.11.0/openshift-apiserver/ds.yaml]: this now also needs @mfojtik's `--v` fix

File: bindata/v3.11.0/openshift-apiserver/ds.yaml [hunk 2]
```diff
@@ -26,9 +26,12 @@ spec:
       - name: openshift-apiserver
         image: ${IMAGE}
         imagePullPolicy: IfNotPresent
-        command: ["hypershift", "openshift-apiserver"]
+        command: ["/bin/bash", "-xec"]
         args:
-        - "--config=/var/run/configmaps/config/config.yaml"
+        - |
+          mkdir -p /var/log/openshift-apiserver
```
Comment (sttts) [inline bindata/v3.11.0/openshift-apiserver/ds.yaml]: we can drop mkdir probably.

**Comment by soltysh** [comment]
This was reworked to be inline with the approach in https://github.com/openshift/cluster-kube-apiserver-operator/pull/317

**Pull request merged**